Lint fix (#972)
demisto/demisto-sdk: CHANGELOG.md

* Added validations that checks in read-me for empty sections or leftovers from the auto generated read-me that should be changed.
* Added new code validation for *NotImplementedError* to raise a warning in `XSOAR-linter`.
* Added validation for support types in the pack metadata file.

> * Fixed a regression in the **lint** command where it was ignoring the -i flag.

Reviewer: ```suggestion
* Fixed an isse in the **lint** command where the *-i* flag was ignored.
```
